package.json

```diff
@@ -16,18 +16,19 @@
     "deploy:all": "yarn deploy:customer1 && yarn deploy:customer2",
     "deploy:all:production": "yarn deploy:customer1:production && yarn deploy:customer2:production",
     "deploy:all:all": "yarn deploy:all && yarn deploy:all:production",
-    "deploy:customer": "yarn vercel:cleanup && vercel --confirm --local-config=vercel.${CUSTOMER_REF}.staging.json --debug",
-    "deploy:customer:production:simple": "yarn vercel:cleanup && vercel --confirm --local-config=vercel.${CUSTOMER_REF}.production.json --prod --debug",
+    "deploy:ci:gha": "yarn vercel:cleanup && echo 'XXX - GIT_COMMIT_REF: ' $GIT_COMMIT_REF && echo 'GIT_COMMIT_SHA: ' $GIT_COMMIT_SHA && yarn vercel:deploy --local-config=vercel.${CUSTOMER_REF}.staging.json",
```

Comment:
@Demmonius I tried to echo here but don't see those printed on any log. Is that normal?

Reply:
Not really, I'm going to have a look !